google-cloud-monitoring/lib/google/monitoring/v3/metric_service_pb.rb

```diff
@@ -93,6 +102,8 @@ module V3
       ListTimeSeriesResponse = Google::Protobuf::DescriptorPool.generated_pool.lookup("google.monitoring.v3.ListTimeSeriesResponse").msgclass
       CreateTimeSeriesRequest = Google::Protobuf::DescriptorPool.generated_pool.lookup("google.monitoring.v3.CreateTimeSeriesRequest").msgclass
       CreateTimeSeriesError = Google::Protobuf::DescriptorPool.generated_pool.lookup("google.monitoring.v3.CreateTimeSeriesError").msgclass
+      CreateTimeSeriesSummary = Google::Protobuf::DescriptorPool.generated_pool.lookup("google.monitoring.v3.CreateTimeSeriesSummary").msgclass
```

Comment:
This is added but not documented and not exposed by any other class. Should we document this?

Reply:
It looks like there are more Monitoring changes since this PR. Should we close this PR to see the recent changes?